src/Service/Importer.php

```diff
@@ -426,11 +429,12 @@ protected function downloadFile($strUrl, $strTargetFolder, $strBase): ?FilesMode
         // determine the filename
         $strFileName = $objPathinfo->filename.'-'.substr(md5($strUrl), 0, 8).'.'.$objPathinfo->extension;
 
-        // determine the full (relative) file path
-        $strFilePath = $strTargetFolder.'/'.$strSubFolder.'/'.$strFileName;
+        // determine file paths
+        $absoluteFilePath = Path::join($this->projectDir, $strTargetFolder.'/'.$strSubFolder.'/'.$strFileName);
+        $strFilePath = Path::makeRelative($absoluteFilePath, $this->projectDir);
```

Comment:
Can you explain why this is necessary? `$strTargetFolder.'/'.$strSubFolder.'/'.$strFileName` is already a relative path.

Reply:
As I already wrote, we got a relative folder, but with double dots in it. So we got `files/media/import/plugin/name/../../wp-content/something`. The `plugin/name` folder did not exist locally and we got an exception from Dbfs class. `Path::join` resolved such paths.

Reply:
I see, but then all you need is 

```php
$strFilePath = Path::join($strTargetFolder, $strSubFolder, $strFileName);
```

Reply:
Ok, that is also an option, but we also need the absolute path. Just a different approach to reach the same thing :)

The alternative would be:
```
$strFilePath = Path::join($strTargetFolder, $strSubFolder, $strFileName);
$absoluteFilePath = Path::join($this->projectDir, strFilePath);
```
Should I adjust it that way?

Reply:
Yes, it's a small performance improvement in any case. Previously the absolute path was always created on the fly whenever needed with `Path::join($this->projectDir, $strFilePath)`. Putting it into a reusable variable makes sense.

Reply:
Adjusted in my last commits.